spring-integration-webflux/src/test/java/org/springframework/integration/webflux/outbound/WebFluxRequestExecutingMessageHandlerTests.java

```diff
@@ -376,4 +386,63 @@ void testMaxInMemorySizeExceeded() {
 				.isEqualTo("Exceeded limit on max bytes to buffer : 1");
 	}
 
+	@Test
+	@SuppressWarnings("unchecked")
+	void testFluxReplyWithRequestAttribute() {
+		ClientHttpConnector httpConnector = new HttpHandlerConnector((request, response) -> {
+			response.setStatusCode(HttpStatus.OK);
+			response.getHeaders().setContentType(MediaType.TEXT_PLAIN);
+
+			DataBufferFactory bufferFactory = response.bufferFactory();
+
+			Mono<DataBuffer> data = Mono.just(bufferFactory.wrap("foo\nbar\nbaz".getBytes()));
+
+			return response.writeWith(data)
+					.then(Mono.defer(response::setComplete));
+		});
+
+		 class AttributeFilter implements ExchangeFilterFunction {
+
+			Optional<Object> attributeValueName;
+
+			 @Override
+			 public Mono<ClientResponse> filter(ClientRequest request, ExchangeFunction next) {
+				 this.attributeValueName = request.attribute("name");
+				 return next.exchange(request);
+			 }
+		 }
+
+		AttributeFilter attributeFilter = new AttributeFilter();
+		WebClient webClient = WebClient.builder()
+				.clientConnector(httpConnector)
+				.filter(attributeFilter)
+				.build();
+
+		String destinationUri = "https://www.springsource.org/spring-integration";
+		WebFluxRequestExecutingMessageHandler reactiveHandler =
+				new WebFluxRequestExecutingMessageHandler(destinationUri, webClient);
+
+		QueueChannel replyChannel = new QueueChannel();
+		reactiveHandler.setOutputChannel(replyChannel);
+		reactiveHandler.setExpectedResponseType(String.class);
+		reactiveHandler.setReplyPayloadToFlux(true);
+		Expression expr = new SpelExpressionParser().parseExpression("{name:{first:'Nikola'}}");
```

Comment:
Just for your knowledge: there is a `FunctionExpression` impl to be able to call as a lamda.
So, this one would be as simple as: `Expression expr = new FunctionExpression<Object>(o -> Map.of("first", "Nikola"))`.
and much faster than SpEL parsing and its evaluation via reflection 😄